charts/k8s-image-swapper/values.yaml

```diff
@@ -35,10 +35,12 @@ podSecurityPolicy:
 
 podAnnotations: {}
 
-podSecurityContext: {}
+podSecurityContext:
```

Comment:
Can you fix this up so it doesn't appear as change?

Reply:
Fixed